lib/core/src/server/preview/iframe-webpack.config.js

```diff
@@ -39,6 +39,7 @@ export default async ({
   presets,
 }) => {
   const dlls = await presets.apply('webpackDlls', []);
+  const logLevel = await presets.apply('logLevel', []);
```

Comment:
Is this so the user can set `logLevel` in `main.js`? what happens when it gets defined in a preset? Is there one level globally, or does it get scoped somehow?

Reply:
The value is a singular string, not an array.

There's 1 setting, whichever is last wins, `main.js` should always be last.

presets should not be setting this I think.